### PR TITLE
Added ThatLowCarbLife

### PR DESCRIPTION
**Pull request by cdthomp1**
@jadkins89 I finally got around to getting this scrapper refactored. Hope its good to go!

### CODE REVIEW

**Comment by cdthomp1** [comment]
@jadkins89 Any update on getting this pulled in? 

Thanks.

**Comment by jadkins89** [comment]
Sorry to leave you hanging here. I've been busy with a new job and settling into a new house so this project hasn't gotten a lot of love lately. I'll try to look at this and merge it in by this weekend.

File: test/constants/thatlowcarblifeConstants.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            '2 tablespoons cream cheese',
+            '1 egg',
+            '1 teaspoon salt',
+            '⅔ cup ranch dressing',
+            '2 cups shredded cheddar',
+            '1 cup chopped cooked chicken',
+            '6 strips bacon, fried and crumbled',
+            '2 tablespoons chopped chives'
+        ],
+        instructions: [
```
Comment (shaniqwa) [inline test/constants/thatlowcarblifeConstants.js:20]: nicer to trim them :)

Comment (cdthomp1) [reply test/constants/thatlowcarblifeConstants.js:20]: Sometimes I give the the 'ol karate chop 🥋

**Review by jadkins89 — APPROVED** [review]

**Pull request merged**